packages/zip/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "type": "module",
   "main": "./index.js",
+  "types": "./index.d.ts",
```

Comment:
Why is this coming back? Can't we add a single `export * from './src/types.js';` in `index.js` (not `.d.ts`)? That's what I did for `marshal` and it seems to work fine.